apps/poker-roll/src/app/startSession.jsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { TextField, Button, Stack } from "@mui/material";
+import ToggleButton from '@mui/material/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+
+const SessionForm = (props) => {
+    const [formState, setFormState] = useState({
+        buyin: "",
+        game: "",
+        casino: "",
+        location: "",
+      });
+
+      const handleChange = (event) => {
+        const { name, value } = event.target;
+    
+        setFormState({
+          ...formState,
+          [name]: value,
+        });
+      };
+    const [alignment, setAlignment] = React.useState('cashgame');
```

Comment:
Is `alignment` a poker term? Kind of a confusing variable name if not.

Reply:
that is what the docs code was, just pulled it over from material component library